examples/go-client/main.go

```diff
@@ -21,10 +21,10 @@ import (
 	"fmt"
 	"log"
 
+	"github.com/google/uuid"
```

Comment:
Would you mind moving formatting changes to a separate commit, so it's easier to know what I can skim read?